kafka-rest/src/main/java/io/confluent/kafkarest/entities/v3/ProduceResponse.java

```diff
@@ -54,6 +54,10 @@
   @JsonInclude(Include.NON_ABSENT)
   public abstract Optional<ProduceResponseData> getValue();
 
+  @JsonProperty("resume_after_ms")
+  @JsonInclude(Include.NON_ABSENT)
+  public abstract Optional<Long> getResumeAfterMs();
```

Comment:
I picked a long as it was smaller to store than an Instant (because this map could end up holding a lot of values if the rate limit is high so I wanted to keep it as small as possible), and it's a number, so I don't have to do String -> number conversion later in the code as that will slow down the flow.